src/test/java/teammates/common/datatransfer/questions/FeedbackTextResponseDetailsTest.java

```diff
@@ -0,0 +1,53 @@
+package teammates.common.datatransfer.questions;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.SanitizationHelper;
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackTextResponseDetails}.
+ */
+public class FeedbackTextResponseDetailsTest extends BaseTestCase {
+    @Test
+    public void testValidateResponseAnswer_sameValues_shouldReturnTrue() {
+        String testAnswer = "Hello World";
+        FeedbackTextResponseDetails feedbackTextResponseDetails = new FeedbackTextResponseDetails(testAnswer);
+        assertEquals(feedbackTextResponseDetails.getAnswer(), "Hello World");
+    }
+
+    @Test
+    public void testValidateResponseAnswer_differentValues_shouldReturnFalse() {
+        String testAnswer = "Hello World";
+        FeedbackTextResponseDetails feedbackTextResponseDetails = new FeedbackTextResponseDetails(testAnswer);
+        assertNotEquals(feedbackTextResponseDetails.getAnswer(), "World Hello");
+    }
+
+    @Test(expectedExceptions = NullPointerException.class)
+    public void testValidateAttributesOfAnswer_nullValue_shouldThrowException() {
```

Comment:
you can assert is null here instead of forcing an NPE

Reply:
I think your naming of the test function should change here as well?